[ENH] Apply derivative definition to qMRI appendix (#1813)
bids-standard/bids-specification: src/appendices/qmri.md

Please visit the [file collections appendix](./file-collections.md#magnetic-resonance-imaging) to see the list of currently supported qMRI applications.

### Outputs are quantitative maps

qMRI maps are stored differently depending on the process that generated them.

> Pre-generated qMRI maps MUST be stored as part of raw BIDS datasets,

Reviewer: ```suggestion
Pre-generated qMRI maps MAY be stored as part of raw BIDS datasets,
```

This is unenforceable and I don't think we should require it, anyway.

Somebody might prefer to create a dataset like:

```
derivatives/
  scanner-generated/
    sub-01/
      anat/
        sub-01_T1map.nii.gz
        sub-01_T1map.json
        sub-01_UNIT1.nii.gz
        sub-01_UNIT1.json
  open-algorithm/
    sub-01/
      anat/
        sub-01_T1map.nii.gz
        sub-01_T1map.json
        sub-01_UNIT1.nii.gz
        sub-01_UNIT1.json
sub-01/
  anat/
    sub-01_inv-1_part-mag_MP2RAGE.nii.gz
    sub-01_inv-1_part-phase_MP2RAGE.nii.gz
    sub-01_inv-1_MP2RAGE.json
    sub-01_inv-2_part-mag_MP2RAGE.nii.gz
    sub-01_inv-2_part-phase_MP2RAGE.nii.gz
    sub-01_inv-2_MP2RAGE.json
```

There the symmetry between the different derivatives would make the dataset more legible.

> where as they MUST be stored in a derivative BIDS dataset if they were post-generated.

Reviewer 2: ```suggestion
whereas they MUST be stored in a derivative BIDS dataset if they were post-generated.
```

> See the example below of a `T1map` generated from an `MP2RAGE` file collection using either options.

Reviewer 2: ```suggestion
See the example below of a `T1map` generated from an `MP2RAGE` file collection using either option.
```

### Outputs are quantitative maps

qMRI maps are stored differently depending on the process that generated them.
Pre-generated qMRI maps MAY be stored as part of a raw BIDS dataset,

> where as it MAY be stored in a derivative BIDS dataset if it were post-generated.

Reviewer: I think this one should be a MUST, even though it's unenforceable. Here the MUST would be best interpreted as a directive to a tool that does the calculation that it should not modify the original dataset.

```suggestion
where as they MUST be stored in a derivative BIDS dataset if they were post-generated.
```

Also, updating pronouns for agreement with "maps".

provided by the stock sequence. Instead, the `magnitude` and `phase` images are exported. Please
see the relevant discussion at [qMRLab issue #255](https://github.com/qMRLab/qMRLab/issues/255).

Therefore, the `UNIT1` image provided by the scanner

> MUST be stored under the `anat` in a raw BIDS dataset

Reviewer: ```suggestion
SHOULD be stored under the `anat` in a raw BIDS dataset
```

along with the `MP2RAGE` file collection
and to be used as the primary input for quantifying a `T1map`.

If an additional `UNIT1` image is calculated offline,

> then the output is to be stored in a derivative BIDS dataset with necessary provenance information.

Author: ```suggestion
then the output MAY be stored in a derivative BIDS dataset with necessary provenance information.
```

@effigies here too?

Reviewer: Here I would say MUST:

```suggestion
then the output MUST be stored in a derivative BIDS dataset with necessary provenance information.
```
